WordPress/Classes/ViewRelated/NUX/SiteCreationDomainSearchTableViewCell.xib

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The search bar doesn't have placeholder text. Per the design, it should be "Type to get more suggestions".